src/aspire/estimation/covar2d.py

```diff
@@ -149,7 +156,8 @@ def get_covar(self, coeffs, ctf_fb=None, ctf_idx=None, mean_coeff=None,
             raise RuntimeError('The coefficients need to be calculated!')
 
         if (ctf_fb is None) or (ctf_idx is None):
-            return self._get_covar(coeffs, mean_coeff, do_refl)
+            ctf_idx = np.zeros(coeffs.shape[1], dtype=int)
+            ctf_fb = [blk_diag_eye(blk_diag_partition(RadialCTFFilter().fb_mat(self.basis)))]
```

Comment:
Again, should be possible to get this from `basis`. Let's make an issue.

Reply:
Made the issue #105.